src/pages/admin/company-profile/components/company-profile-requisites-tab/CompanyProfileRequisitesTab.module.scss

```
@@ -0,0 +1,25 @@
+    gap: 24px;
+    width: 660px;
+    max-width: 660px;
+}
+
+.form-row {
+    width: 100%;
+}
+
+.requisites-tab-title {
+    @include type.h4-semibold;
+    color: c.$blue-900;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)